packages/ruleset-migrator/src/utils/__tests__/isPackageImport.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { isPackageImport } from '../isPackageImport';
+
+describe('isPackageImport util', () => {
+  it.each([
```

Comment:
Can we also have a test case here for a versioned package? the docs specify that we should be able to pin versions

Reply:
This is not how pinning works though.
Overall there are 2 ways to pin a version:
- if you are a CLI, the version of the package you have installed will be preferred. This means the version itself is specified in `package.json` and no in the ruleset.
- regardless of the type of a user, one can always try to use services like https://unpkg.com/, but their have their own way of doing that and this wouldn't be considered as a package import, since it's just an external ruleset we fetch

Reply:
If it's not possible [as per the current docs](https://github.com/stoplightio/spectral/blob/develop/docs/guides/7-sharing-rulesets.md?plain=1#L119-L124) can we amend them please?

Reading them, I'd expect to be able to i.e. just run the below without a `package.json` to specify the version of the pinned dependencies:

```sh
npx @stoplight/spectral-cli lint v1alpha/openapi.yml
```

Reply:
> If it's not possible as per the current docs can we amend them please?

that bit is indeed outdated.
Going to update it.